src/pages/class/[cid]/question/[qid]/solve.tsx

```diff
@@ -116,9 +117,9 @@ export default function Page() {
   const reportSubmit = useCallback(
     async (msg: string) => {
       toggleModal()
-      // TODO: Link to the backend
+      qid && (await request<SubmitReportParams, SubmitReportResults>(`question/submitReport`, { qid, comment: msg }))
```

Comment:
for better readability
```suggestion
      if (qid) {
        await request<SubmitReportParams, SubmitReportResults>(`question/submitReport`, { qid, comment: msg })
      }
```

Reply:
> for better readability

updated here https://github.com/kixlab/kuiz/pull/85/commits/d134e5fe14571b9bdb1ec0568dfd4c15b5e241d1 :)